content/languages/python/authoring/index.md

```diff
@@ -56,9 +56,24 @@ Some kata should not be translated into Python, because it can be difficult to k
 
 ## Coding
 
+
+### Code style
+
 Python code should stick to generally recognized Python conventions, with [PEP-8](https://www.python.org/dev/peps/pep-0008/) being most widely accepted.
 
 
+### Imports
+
+It is strongly recommended to import solution modules explicitly. Possible modules are:
+- `solution`, containing user's submitted solution,
+- `preloaded`, containing preloaded code,
+- `codewars_test`, containing [Codewars Python testing framework](/languages/python/codewars-test/).
+
+When any of these imports is missing, then, for compatibility reasons, it will be automatically added by Codewars preprocessor when solution is built. However, relying on this behavior is discouraged, as it's considered deprecated and might change in the future.
```

Comment:
```suggestion
When any of these imports are missing, then, for compatibility reasons, it will be automatically added by Codewars preprocessor when the solution is built. However, relying on this behavior is discouraged, as it's considered deprecated and might change in the future.
```